pkg/image/registry/imagestreamtag/rest.go

```diff
@@ -93,12 +135,54 @@ func (r *REST) Get(ctx kapi.Context, id string) (runtime.Object, error) {
 			Namespace:         kapi.NamespaceValue(ctx),
 			Name:              id,
 			CreationTimestamp: event.Created,
+			Annotations:       istagAnnotations,
+
+			// resource version is logically the later of image or imagestream
+			// with numbers, concatenating the two will alway produce a number larger than either one
+			// with strings, it will produce something unique.
+			ResourceVersion: stream.ResourceVersion + image.ResourceVersion,
```

Comment:
Return the image stream resource version for now.  Image resource version is ommited and hidden.